content/en/docs/configuring-sdk/custom-http.md

```diff
@@ -1,21 +1,54 @@
 ---
-title: "Creating a Custom HTTP Client"
-linkTitle: "Custom HTTP Client"
+title: "Customizing the HTTP Client"
+linkTitle: "HTTP Client"
+date: "2020-11-12"
 description: "Create a custom HTTP client with the AWS SDK for Go V2 to specify custom timeout values."
 ---
 
-The AWS SDK for Go V2 uses a default HTTP client with default configuration values. Although you can change some of
-these configuration values, the default HTTP client and transport are not sufficiently configurable for customers using
-the AWS SDK for Go V2 in an environment with high throughput and low latency requirements. This section describes how to create a
-custom HTTP client, and use that client to create AWS SDK for Go V2 calls.
+The {{% alias sdk-go %}} uses a default HTTP client with default configuration values. Although you can change some of
+these configuration values, the default HTTP client and transport are not sufficiently configured for customers using
+the {{% alias sdk-go %}} in an environment with high throughput and low latency requirements. This section describes how
+to configure a custom HTTP client, and use that client to create {{% alias sdk-go %}} calls.
 
-To assist you in creating a custom HTTP client, this section describes how to create a structure to encapsulate the
-custom settings, create a function to create a custom HTTP client based on those settings, and use that custom HTTP
-client to call an AWS SDK for Go V2 service client.
+To assist you in creating a custom HTTP client, this section describes how to the 
+[aws.BuildableHTTPClient]({{< apiref "aws#BuildableHTTPClient" >}}) to configure custom settings, and use that client
+with an {{% alias sdk-go %}} service client.
 
 Let's define what we want to customize.
 
-## Dialer.KeepAlive
+
+## Overriding During Configuration Loading
+Custom HTTP clients can be provided when calling [LoadDefaultConfig]({{< apiref "config#LoadDefaultConfig" >}}) by
+wrapping the client using [WithHTTP]({{< apiref "config#WithHTTP" >}}) and passing the resulting value to 
+`LoadDefaultConfig`. For example to pass `customClient` as our client:
+
+```go
+cfg, err := config.LoadDefaultConfig(config.WithHTTP(customClient))
+```
```

Comment:
nit `WithHTTPClient` instead of `WithHTTP`

Reply:
Good catch copy and paste failure.